test/models/release_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 3
+SingleCov.covered! uncovered: 4
```

Comment:
Could you add a test for `code_authors`?

Reply:
I'd rather just remove it then. There's 0 logic in it.